MAD.Integration.Common/Jobs/RescheduleJobByDateOnExceptionAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Hangfire.Common;
+using Hangfire.States;
+using System;
+
+namespace MAD.Integration.Common.Jobs
+{
+    public class RescheduleJobByDateOnExceptionAttribute : JobFilterAttribute, IElectStateFilter
```

Comment:
Rename this to RescheduleJobByDateFilter and don't inherit from JobFilterAttribute.
Then apply this as a default JobFilter inside the IntegrationHost.CreateDefaultBuilder

Reply:
Very well bruv'nor